src/main/java/com/flansmod/common/teams/GameType.java

```diff
@@ -128,6 +132,42 @@ public boolean playerCanAttack(EntityPlayerMP attacker, Team attackerTeam, Entit
      * Called when any entity is killed. This allows one to track mob deaths too
      */
     public void entityKilled(Entity entity, DamageSource source) {
+
+    }
+
+    public void vehicleDestroyed(EntityDriveable driveable2, EntityBullet bullet){
+        if (bullet!=null && driveable2!=null) {
+            if (bullet.owner != null) {
+                EntityPlayerMP attacker = (EntityPlayerMP) bullet.owner;
+                EntityDriveable driveable = driveable2;
+//                if(driveable.riddenByEntity!=null &&
+//                        driveable.riddenByEntity instanceof EntityPlayer &&
+//                        !getPlayerData((EntityPlayerMP) driveable.riddenByEntity).team.equals(getPlayerData(attacker).team)) {
+                    if(true){
+                        getPlayerInfo(attacker).vehiclesDestroyed++;
+                        FlansMod.log("Minigames: "+driveable.getDriveableType().name + " destroyed by "+attacker+" with using "+bullet.firedFrom.name);
```

Comment:
This should probably be removed too, I don't think it's a good idea to log every vehicle death.

Reply:
okay